crates/noirc_evaluator/src/ssa_refactor/ir/dfg.rs

```diff
@@ -340,6 +346,17 @@ impl DataFlowGraph {
         }
     }
 
+    /// Returns the Type::Array associated with this ValueId if it refers to an array parameter.
+    /// Otherwise, this returns None.
+    pub(crate) fn get_array_parameter(&self, value: ValueId) -> Option<(Rc<CompositeType>, usize)> {
```

Comment:
Lets rename this to `get_array_parameter_type` since it returns the type of the parameter rather than the parameter itself.

Reply:
Done